tools/Nethermind.Tools.Kute/ResponseTracer/FileResponseTracer.cs

```diff
@@ -5,20 +5,15 @@
 
 namespace Nethermind.Tools.Kute.ResponseTracer;
 
-public class FileResponseTracer : IResponseTracer
+public class FileResponseTracer(string tracesFilePath) : IResponseTracer
```

Comment:
No need to change this file at all: null check is performed in `Program.cs` and `tracesFilePath` is never null.